### ISSUE TITLE
Awesome papers and resources

### ISSUE THREAD

**Issue by zdx3578** [HyperGAN/HyperGAN#1]
beta-VAE is also very good ref : http://openreview.net/forum?id=Sy2fzU9gl

Learning an interpretable factorised representation of the independent data gen- erative factors of the world without supervision is an important precursor for the development of artificial intelligence that is able to learn and reason in the same way that humans do. We introduce  -VAE, a new state-of-the-art framework for automated discovery of interpretable factorised latent representations from raw image data in a completely unsupervised manner. Our approach is a modification of the variational autoencoder (VAE) framework. We introduce an adjustable hy- perparameter   that balances latent channel capacity and independence constraints with reconstruction accuracy. We demonstrate that  -VAE with appropriately tuned   > 1 qualitatively outperforms VAE (  = 1), as well as state of the art unsu- pervised (InfoGAN) and semi-supervised (DC-IGN) approaches to disentangled factor learning on a variety of datasets (celebA, faces and chairs). Furthermore, we devise a protocol to quantitatively compare the degree of disentanglement learnt by different models, and show that our approach also significantly outperforms all baselines quantitatively. Unlike InfoGAN,  -VAE is stable to train, makes few assumptions about the data and relies on tuning a single hyperparameter  , which can be directly optimised through a hyperparameter search using weakly labelled data or through heuristic visual inspection for purely unsupervised data.

and

https://arxiv.org/abs/1606.05579

Early Visual Concept Learning with Unsupervised Deep Learning

Automated discovery of early visual concepts from raw image data is a major open challenge in AI research. Addressing this problem, we propose an unsupervised approach for learning disentangled representations of the underlying factors of variation. We draw inspiration from neuroscience, and show how this can be achieved in an unsupervised generative model by applying the same learning pressures as have been suggested to act in the ventral visual stream in the brain. By enforcing redundancy reduction, encouraging statistical independence, and exposure to data with transform continuities analogous to those to which human infants are exposed, we obtain a variational autoencoder (VAE) framework capable of learning disentangled factors. Our approach makes few assumptions and works well across a wide variety of datasets. Furthermore, our solution has useful emergent properties, such as zero-shot inference and an intuitive understanding of "objectness".

**Comment by zdx3578** [comment]
http://www.evolvingai.org/ppgn

**Comment by zdx3578** [comment]
https://github.com/soumith/ganhacks

**Comment by martyn** [comment]
https://arxiv.org/pdf/1612.03242v1.pdf

**Comment by martyn** [comment]
https://github.com/andersbll/autoencoding_beyond_pixels

**Comment by zdx3578** [comment]
Wasserstein GAN  https://github.com/tdeboissiere/DeepLearningImplementations/blob/master/WassersteinGAN/README.md 
We introduce a new algorithm named WGAN, an alternative to traditional GAN training. In this new model, we show that we can improve the stability of learning, get rid of problems like mode collapse, and provide meaningful learning curves useful for debugging and hyperparameter searches. Furthermore, we show that the corresponding optimization problem is sound, and provide extensive theoretical work highlighting the deep connections to other distances between distributions.

**Comment by zdx3578** [comment]
https://github.com/wayaai/SimGAN/blob/master/sim-gan.py#L138
    # define custom local adversarial loss (softmax for each image section) for the discriminator
    # the adversarial loss function is the sum of the cross-entropy losses over the local patches
    #
    def local_adversarial_loss(y_true, y_pred):
        # y_true and y_pred have shape (batch_size, # of local patches, 2), but really we just want to average over
        # the local patches and batch size so we can reshape to (batch_size * # of local patches, 2)
        y_true = tf.reshape(y_true, (-1, 2))
        y_pred = tf.reshape(y_pred, (-1, 2))
        loss = tf.nn.softmax_cross_entropy_with_logits(labels=y_true, logits=y_pred)

        return tf.reduce_mean(loss)

**Comment by zdx3578** [comment]
https://github.com/dribnet/plat   http://arxiv.org/abs/1609.04468
Utilities for exploring generative latent spaces as described in the Sampling Generative Networks paper.

paper:https://arxiv.org/abs/1701.04722
https://gist.github.com/poolio/b71eb943d6537d01f46e7b20e9225149

**Comment by zdx3578** [comment]
https://github.com/affinelayer/pix2pix-tensorflow

**Comment by zdx3578** [comment]
Guo-Jn Qi. Loss-Sensitive Generative Adversarial Networks on Lipschitz Densities. arXiv:1701.06264 [pdf]

The cost function used in this GLS-GAN implementation is a leaky rectified linear unit with a slope set in input opt. By default it is 0.2.

If you set slope to 0, you shall get LS-GAN;
If you set slope to 1.0, you shall get WGAN.


https://github.com/guojunq/glsgan

**Comment by martyn** [comment]
https://arxiv.org/abs/1611.04076v2

Least Squares Generative Adversarial Networks 

Not to be confused with GLS-GAN or Loss Sensitive GAN above.

**Comment by martyn** [comment]
DiscoGAN
https://arxiv.org/pdf/1703.05192.pdf
"[...] we address the task of discovering cross-domain relations given unpaired data[...]"

**Comment by martyn** [comment]
CycleGAN
https://junyanz.github.io/CycleGAN/

**Comment by martyn** [comment]
BEGAN
https://arxiv.org/pdf/1703.10717.pdf
"We propose a new equilibrium enforcing method paired with a loss derived from
the Wasserstein distance for training auto-encoder based Generative Adversarial
Networks"

**Comment by zdx3578** [comment]
https://github.com/junyanz/CycleGAN

**Comment by zdx3578** [comment]
https://github.com/GunhoChoi/DiscoGAN_TF
https://github.com/carpedm20/DiscoGAN-pytorch
https://github.com/SKTBrain/DiscoGAN

**Comment by IgorSusmelj** [comment]
**ABC-GAN**: Adaptive Blur and Control for improved training stability of Generative Adversarial Networks

- We present two methods only affecting the way you train a GAN to improve it's stability, convergence speed, generated image quality and output resolution.

- We use an adaptive controller (similar to the one used in BEGAN) for improved training speed

- We use an adaptive blur for assisting the discriminator during training by shifting the focus from higher frequency signals to lower frequency signals. We noticed that discriminators tend to be too sensitive on the generated images and using a blur we focus more on the overall structure (e.g. of a face) and not the individual pixel colors.

We implemented our two approaches on top of DCGAN. It would be cool to see it used on other networks. 
https://github.com/IgorSusmelj/ABC-GAN

**Comment by zdx3578** [comment]
https://github.com/LMescheder/AdversarialVariationalBayes     Adversarial Variational Bayes:
Unifying Variational Autoencoders and Generative Adversarial Networks

**Comment by martyn** [comment]
This thread collected useful references for beta-VAE/latent exploration, WGAN, pix2pix, CycleGAN/DiscoGAN and BEGAN. We are preserving it as a research index and closing the omnibus issue as superseded by the resurrection plan’s recipe-admission process. Each future addition needs a named application, source/license review, reproducible quality and resource evidence, and tested training/deployment profiles. That requirement is retained; this closure does not claim these papers are all implemented. The [current plan](https://github.com/HyperGAN/HyperGAN/blob/develop/reports/resurrecting-hypergan-plan-2026-09-18.md) explains the admission gates.

Audit record: [individual disposition and retained work](https://github.com/HyperGAN/HyperGAN/blob/f4053ae1/reports/issue-audit-2026-09-18.md), published in #302.
